portfolio/src/main/java/com/google/sps/data/CalendarEvent.java

```diff
@@ -47,4 +47,12 @@ public Instant getStartTimeInstant() {
   public Instant getEndTimeInstant() {
     return endTime;
   }
+
+  public long getStartTimeLong() {
```

Comment:
Consider clarifying the time unit in the method name.

ex: getStartTimeSecondsLong (or just getStartTimeSeconds)

Reply:
By the way, do we really need this method? I think the callers can choose to invoke getEpochSecond themselves. I think that is one major benefit of returning Instant; the callers can decide how to work with the instant.

Reply:
I removed the methods.